addons/website_sale_loyalty/tests/test_shop_loyalty_payment.py

```diff
@@ -29,7 +29,7 @@ def test_expired_reward_validation(self):
         order = self.empty_order
         program = self.program_gift_card
 
-        program.date_to = date.today()  # set program to expire after today
+        program.date_to = date.today() + timedelta(days=1) # set program to expire after tomorrow
```

Comment:
```suggestion
        program.date_to = date.today() + timedelta(days=1)  # set program to expire after tomorrow
```

... Saw it when sending the review, can you just readd the double space before a comment? Thanks :pray:

Reply:
yes for sure, thank you very much for the review